kaizen/runner.py

```
@@ -0,0 +1,461 @@
+        # Anthropic API key is optional - only warn if not present
+        if not os.getenv('ANTHROPIC_API_KEY'):
+            console.print("[yellow]Note: ANTHROPIC_API_KEY not found - Anthropic-based tests will use OpenAI instead[/yellow]")
+
+    def load_test_file(self, file_path: str) -> Dict:
+        """Load and parse a YAML test file."""
+        try:
+            with open(file_path, 'r') as f:
+                return yaml.safe_load(f)
+        except yaml.YAMLError as e:
+            console.print(f"[red]Error: Invalid YAML syntax in {file_path}[/red]")
+            console.print(f"Line {e.problem_mark.line + 1}: {e.problem}")
```

Pylint (3.3.7)

[error] Instance of 'YAMLError' has no 'problem_mark' member

(E1101)